src/App.css

```diff
@@ -0,0 +1,65 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Segoe UI", Arial, sans-serif;
+  line-height: 1.4;
+  color: #444;
+  background: #fff;
+  height: 100vh;
+}
+
+.Todo {
```

Comment:
Great job in following the cascading order of their order, also there is a best practice to follow for naming ``CSS`` variables and that is to name the attributes and variables always with small letters. ☺️